pom.xml

```diff
@@ -91,6 +94,12 @@
         </dependency>
 
         <dependency>
+            <groupId>org.xerial</groupId>
```

Comment:
This is duplicated between `runtime` and `test` which causes maven to complain with a warning when you run it. The `runtime` one should cover the `test` classpath as well, so I think you just want to remove this second copy.